Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+const Thumbnail = props => {
+	return (
+		<div  className="thumbnail">
+			<img className="crest" src={require('../images/lambdacrest.png')} />
+   	</div>
```

Comment:
same here...require is old syntax. But don't forget it because we'll use it in backend :)